main/acle.md

```diff
@@ -3613,6 +3614,80 @@ values.
 | KEEP                 | 0         | Temporal fetch of the addressed location (that is, allocate in cache normally) |
 | STRM                 | 1         | Streaming fetch of the addressed location (that is, memory used only once)     |
 
+The `__ARM_PREFETCH_RANGE` macro can be used to test for the presence of the
+following range prefetch intrinsics:
+
+``` c
+  void __pldx_range(/*constant*/ unsigned int /*access_kind*/,
+                    /*constant*/ unsigned int /*retention_policy*/,
+                    /*constant*/ size_t       /*reuse distance*/,
+                    /*constant*/ signed int   /*stride*/,
+                    /*constant*/ unsigned int /*count*/,
+                    /*constant*/ signed int   /*length*/,
```

Comment:
Not a strongly held view, but after reading the description of `__pld_range` I do feel its "parameters" have a more natural order (i.e. length, count, stride, reuse distance). What do you think to using the same order for `__pldx_range`?